docs/getting-started/install-endpoint.asciidoc

```diff
@@ -131,15 +141,51 @@ Behavioral ransomware prevention detects and stops ransomware attacks on Windows
 
 Ransomware protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license]. If you upgrade to a Platinum+ license from Basic or Gold, ransomware protection will be disabled by default.
 
-Ransomware protection levels are as follows:
+Ransomware protection levels are:
 
-* **Detect**: Detects ransomware on the host and generates an alert. The {agent} will **not** block malware. Select the **Notify User** option to enable user notifications.
+* **Detect**: Detects ransomware on the host and generates an alert. The {agent} will **not** block ransomware. Select the **Notify User** option to enable user notifications.
 * **Prevent** (Default): Detects ransomware on the host, blocks it from executing, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
 +
 TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.
 
 [role="screenshot"]
-image::images/install-endpoint/ransomware-protection.png[]
+image::images/install-endpoint/ransomware-protection.png[Detail of ransomware protection section.]
+
+[discrete]
+[[memory-protection]]
+=== Memory manipulation protection
```

Comment:
We renamed this to "Memory threat protection" in a later BC: 
![image](https://user-images.githubusercontent.com/42078554/133128674-27eda368-d089-4d66-a16b-e1350eac9130.png)

Reply:
Thanks for the heads-up! Looks like "Behavior Protection" has also been renamed to "Malicious behavior," so I'll make those updates, too.